src/helpers/generateCommunityProposal.ts

```diff
@@ -49,26 +48,26 @@ const generateCommunityProposal = ( community: Community, contract: Contract ) =
     const { ambassadorAddress, requestByAddress, id, name, description } = community;
     const { claimAmount, maxClaim, baseInterval, incrementInterval } = contract;
     const { minTranche, maxTranche } = getTranches();
-    const { maxBeneficiaries, exponential } = PROPOSAL_CONSTANTS;
+    const { maxBeneficiaries, decreaseStep } = PROPOSAL_CONSTANTS;
 
     return {
         ambassador: ambassadorAddress,
         baseInterval,
-        claimAmount,
-        decreaseStep: toToken(0.01),
+        claimAmount: claimAmount.toString(),
+        decreaseStep: (decreaseStep).toString(),
         incrementInterval,
         managers: [requestByAddress],
         maxBeneficiaries,
-        maxClaim,
-        maxTranche: toToken(maxTranche, { EXPONENTIAL_AT: exponential }),
-        minTranche: toToken(minTranche),
+        maxClaim: maxClaim.toString(),
+        maxTranche: maxTranche.toString(),
+        minTranche: minTranche.toString(),
         proposalDescription: `
         ## Description:
         ${description}
 
         UBI Contract Parameters:
-        Claim Amount: ${toNumber(claimAmount)}
-        Max Claim: ${toNumber(maxClaim)}
+        Claim Amount: ${claimAmount}
+        Max Claim: ${maxClaim}
         Base Interval: ${frequencyToText(+baseInterval)}
         Increment Interval: ${incrementInterval} minutes
```

Comment:
```suggestion
        Increment Interval: ${incrementInterval / 12} minutes
```
taking the opportunity to fix an issue. `incrementInterval` is in blocks, we want minutes. There are 12 blocks per minute on Celo blockchain (for context).

Reply:
Addressed the comments and replaced in the EditPending.tsx as well, Thanks.